src/vmm/src/persist.rs

```diff
@@ -32,10 +32,12 @@ use seccompiler::BpfThreadMap;
 use snapshot::Snapshot;
 use versionize::{VersionMap, Versionize, VersionizeResult};
 use versionize_derive::Versionize;
+use virtio_gen::virtio_ring::VIRTIO_RING_F_EVENT_IDX;
 use vm_memory::GuestMemoryMmap;
 
 #[cfg(target_arch = "x86_64")]
 const FC_V0_23_SNAP_VERSION: u16 = 1;
+const FC_V0_26_SNAP_VERSION: u16 = 4;
```

Comment:
how does the version encoding numbering work? just looking at this code here it looks like we're jumping from version `1` to version `4`.. What am I missing, and how can we make this clearer for future PRs?

(this comment can also be addressed in a separate PR with its own issue)

Reply:
Fixed.